### PR TITLE
fix ThrownExpBottleMockTest

### PR DESCRIPTION
**Pull request by Nirbose**
# Description

# Checklist
The following items should be checked before the pull request can be merged.
- [x] Code follows existing style.
- [x] Unit tests added (if applicable).

### CODE REVIEW

File: extra/TestPlugin/build.gradle.kts [hunk 1]
```diff
@@ -9,3 +9,7 @@ repositories {
 dependencies {
 	compileOnly("io.papermc.paper:paper-api:${rootProject.property("paper.api.full-version")}")
 }
+java {
+    sourceCompatibility = JavaVersion.VERSION_19
+    targetCompatibility = JavaVersion.VERSION_19
+}
```
Comment (thelooter) [inline extra/TestPlugin/build.gradle.kts]: why was this added

**Review by thelooter — CHANGES_REQUESTED** [review]

**Review by thelooter — APPROVED** [review]

File: extra/TestPlugin/build.gradle.kts (hunk 1 above)
Comment (Nirbose) [reply extra/TestPlugin/build.gradle.kts]: Too fast, it's a mistake on my part, I didn't see it during the push, sorry
